src/test/api/karate/cards/cardsReminder.feature

```diff
@@ -32,46 +32,122 @@ Feature: CardsReminder
     ]
     """
 
-Scenario: ResetCardsReadsAndAcks
+    
+  * def entity1Array =
+      """
+      [   "ENTITY1_FR"
+      ]
+      """
+
+            * def entity2Array =
+      """
+      [   "ENTITY2_FR"
+      ]
+      """
+
+Scenario: Send card and receive reminder
+
+
+#Create new perimeter
+* callonce read('../common/createPerimeter.feature') {perimeter: '#(perimeter)', token: '#(authTokenAdmin)'}
+
+#Attach perimeter to group
+    Given url opfabUrl + 'users/groups/ReadOnly/perimeters'
+    And header Authorization = 'Bearer ' + authTokenAdmin
+    And request perimeterArray
+    When method patch
+    Then status 200
+
+
+    * def currentDate = new Date()
+    * def now = new Date().valueOf()
+    * def yesterday = new java.math.BigDecimal(now - 8640000)
+    * def tomorrow = new java.math.BigDecimal(now + 8640000)
+    * def hours = new java.lang.Integer(new Date().getHours())
+    * def minutes = new java.lang.Integer(new Date().getMinutes() + 2)
+    * def seconds = new java.lang.Integer(new Date().getSeconds())

```

Comment:
I think it s gone to create problems particularly when the current time is close to the end of the hour (e.g., 10:59). In such cases, adding 2 minutes would result in a time with 61 minutes, which is not valid.

I suggest to first create a date + 2 min ( new Date().valueOf() + `2*60*1000` )
and then extract hours/minutes/seconds from this new object